io_ogre/ui/export.py

```diff
@@ -354,8 +389,16 @@ def execute(self, context):
     # Logging
     EX_Vx_EXPORT_ENABLE_LOGGING = BoolProperty(
         name="Write Exporter Logs",
-        description="Write Log file to the output directory",
+        description="Write Log file to the output directory (blender2ogre.log)",
         default=config.get('EXPORT_ENABLE_LOGGING'))
+    #EX_Vx_DEBUG_LOGGING = BoolProperty(
+    #    name="Debug Logging",
+    #    description="Whether to show DEBUG log messages",
+    #    default=config.get('DEBUG_LOGGING'))
+    EX_Vx_SHOW_LOG_NAME = BoolProperty(
```

Comment:
I dont think this should be an option exported in the UI

Reply:
The option is useful to know from which .py module/file a certain log entry comes.

It could be false by default, but I think it has its use.

In fact, the previous iteration of these logging changes are already showing that info.

Reply:
I agree that this is useful - I just dont think that should be a user-facing option.

As a user, if I want to export something from blender, I dont care about the log and dont want to be bothered about it.

Reply:
OK, I understand

Reply:
Made the changes requested, I don't know how to do a #define in python so I just left it commented.

On the other hand also added a small fix for the Ogre Meshy code (meshy.py) to catch a situation where the Ogre Meshy executable is not present.